stackcopy.py

```
@@ -933,21 +1060,39 @@ def main():
+                        for ft, fi in stem_files_for_dest.items():
+                            old = fi['basename']
+                            new = chosen.get(ft, old)
+                            if new != old:
+                                changes.append((old, new))
+                                fi['basename'] = new
+                        key = (dest_dir_import, stem)
+                        if key not in collision_notified:
+                            collision_notified.add(key)
+                            print_collision_rename_notice(dest_dir_import, stem, changes, args.dry_run)
+
+                    for file_type, file_info in files:
```

Ruff (0.14.13)

Loop control variable `file_type` not used within loop body

(B007)